### PR TITLE
Shubhamdevgupta patch

### PR DESCRIPTION
**Pull request by shubhamdevgupta**
@rahuldevgarg  i have updated the icon please check and merge it

### CODE REVIEW

File: frontend/mgramseva/lib/env/app_config.dart [hunk 1]
```diff
@@ -45,8 +45,8 @@ Future<Map<String, dynamic>> get devConstants async {
     }
     return {
       // _baseUrl: "https://naljalseva.jjm.gov.in/" + state + (state.isNotEmpty?"/":''),
-      // _baseUrl: "https://naljal-uat.digit.org/" + state + (state.isNotEmpty?"/":''),
-      _baseUrl: window.location.origin + "/" + state + (state.isNotEmpty?"/":''),
+       _baseUrl: "https://naljal-uat.digit.org/" + state + (state.isNotEmpty?"/":''),
```
Comment (rahuldevgarg) [inline frontend/mgramseva/lib/env/app_config.dart:48]: revert this change

**Review by rahuldevgarg — CHANGES_REQUESTED** [review]
resolve the comments

File: frontend/mgramseva/lib/env/app_config.dart (hunk 1 above)
Comment (shubhamdevgupta) [reply frontend/mgramseva/lib/env/app_config.dart]: Resolved

**Comment by shubhamdevgupta** [comment]
Resolved Check now and merge
